roboprop_client/views.py

```diff
@@ -97,6 +100,64 @@ def _get_model_details(result):
     }
 
 
+def __remove_outliers_and_sort(items):
+    # Remove single occurences as is most likely an outlier
+    items = [item for item in items if items.count(item) > 1]
+    # Sort by most occurences
+    sorted(items, key=lambda x: items.count(x), reverse=True)
+    # Remove duplicates
+    items = list(set(items))
+    return items
+
+
+def __detect_thumbnail_details(thumbnail):
```

Comment:
How long does  a typical rekognition call lasts?

Reply:
3 or 4 seconds typically. A noticeable hang, but I think acceptable with a spinner (to be done)

Reply:
so 3 seconds * number of thumbnails, right?
Still acceptable. The alternative, but maybe for later as it could be tricky, would be to spin async tagging processes

Reply:
Sorry 3 or 4 seconds with 5 thumbnails, and including upload time, so its not that bad

Reply:
Indeed!